lib/ansible/modules/crypto/openssl_pkcs12.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: openssl_pkcs12
+author: "Guillaume Delpierre (@gdelpierre)"
+version_added: "2.4"
+short_description: Generate OpenSSL PKCS#12 archive.
+description:
+    - This module allows one to (re-)generate PKCS#12.
+requirements:
+    - python-pyOpenSSL
```

Comment:
I don't know the 'minimum' version, but I've tried this module with `0.13.1` - whcih is not on the github of the project[1] since its quite old, but it is the one provided by Cent OS 7 and it worked. Debian 7 ships `0.13.2`. Hence I assumed just specifying `python-pyOpenSSL` would have been enough here.

[1] https://github.com/pyca/pyopenssl/tags